### PR TITLE
Show 3rd party widgets as licensed

### PR DESCRIPTION
**Pull request by fjvallarino**
I was able to add a couple of widgets on testing, but I left the rest only for production. One of the widgets does not have an assigned image (StockTrack Rankings), but that's something that's already happening in the current version.

@alex-deaconu please review

### CODE REVIEW

**Comment by fjvallarino** [comment]
@alex-deaconu updated

File: web/scripts/editor/services/svc-products-factory.js [hunk 1]
```diff
@@ -15,6 +15,191 @@ angular.module('risevision.editor.services')
     'paymentTerms': 'Subscription',
     'trialPeriod': 14,
     'productCode': 'd3a418f1a3acaed42cf452fefb1eaed198a1c620'
+  }, {
```
Comment (alex-deaconu) [inline web/scripts/editor/services/svc-products-factory.js]: @fjvallarino maybe we should split these out into their own file as well? Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino thanks for updating. PR looks good 👍

**Comment by alex-deaconu** [comment]
@fjvallarino looks good 👍 thanks for updating.

**Pull request merged**